README.md

```diff
@@ -6,195 +6,200 @@
 
-Currently supports **FAT-0** & **FAT-1** token standards.
+Currently supports **FAT-0** token standard.
 
 
 
 ## Installation
 
 package.json:
 
-```
+```json
 "dependencies":{
-	"fat-js": "git+https://<github_token>:x-oauth-basic@github.com/DBGrow/fat-js.git"
+	"fat-js": "0.0.0"
 }
 ```
 
-or include directly on project path via clone
-
-
-
-## FAT-0
-
-### Initialization
+or
 
-```javascript
-const {FAT0} = require('fat-js/fat0/FAT0');
-
-//Get the token for FAT-0 token with ID: AQQW
-let testToken = await new FAT0('AQQW');
+CLI:
 
-//after the constructor has been completed the token is synced and ready to go
 ```
-
-
-
-### Get Issuance Entry
-
-```javascript
-let issuance = testToken.getIssuance();
-
-console.log(JSON.stringify(issuance,undefined,2)) 
-/*=>
-{
-  "type": "FAT-0",
-  "issuer": "888888dd9e60c1f0216f753caf5c9b5be4c9ca69db27a6c33d30dce3fe5ee709",
-  "supply": 10000000,
-  "name": "Example Token",
-  "symbol": "EXT",
-   "salt": "874220a808090fb736f345dd5d67ac26eab94c9c9f51b708b05cdc4d42f65aae",
-  "idNonce": "cc4fee39dcdcfc1999ab07689230321acdd83fcd0ace521107041ef354b9cfb5",
-  "idSignature": "33467893a440561d96ae27798dc8be291e1ce264d3c6f36f33a0d983e745f1d87db61c77946fe57db3e185f548d51da85106dfec592383a556091dd45f384b0c"
-}
-*/
+npm install fat-js
 ```
 
 
 
-### Get A Transaction by EntryHash
+## Browser Bundle
 
-```javascript
-  let tx = testToken.getTransaction('e1a71b335c3be54659f84e0d36c6c53d0a7e06a960f1cf5fef3af7faac413f2f');
+A browser friendly bundle of the current fat-js version can be found at `browser/bundle.js`
 
-console.log(JSON.stringify(issuance, undefined, 2)) 
-/*=>
-{
-  "input": {
-    "address": "FA3aECpw3gEZ7CMQvRNxEtKBGKAos3922oqYLcHQ9NqXHudC6YBM",
-    "amount": 100
-  },
-  "output": {
-    "address": "FA3aECpw3gEZ7CMQvRNxEtKBGKAos3922oqYLcHQ9NqXHudC6YBM",
-    "amount": 100
-  },
-  "rcd": "011eddd2b3cfb40c581c2afdf675c60a39baeab4cbe75e2ff8b0dc808de6a67cfe",
-  "txSalt": 88395950515565,
-  "txNonce": 3119868535,
-  "marshalBinarySig": "020000b9f56a7701010064d381a44afa76ad5ab8f0d82adb27d6ffa031302cc3bb281c86b1b3dfabde395e64d381a44afa76ad5ab8f0d82adb27d6ffa031302cc3bb281c86b1b3dfabde395e",
-  "signature": "8d4091f55eb6383514cbc569ab174469d7c4b17929bfe87a8d7879536c2eb3b1e446d0d97eb6267014da4630117c093cb0f72641b1c370351899509ed019e80c",
-  "timestamp": 1536612540,
-  "entryHash": "e1a71b335c3be54659f84e0d36c6c53d0a7e06a960f1cf5fef3af7faac413f2f"
-}
-*/
+To build the bundle from source ensure you have browserify installed, and simply navigate to the root of the fat-js project and run:
+
+```bash
+npm run build
 ```
 
+A fresh `bundle.js` will be built in the  `browser` directory.
+
 
 
-### Get All Transactions
+## Instantiate FAT RPC
 
 ```javascript
- let transactions =
-                testToken.getTransactions();
+const RPCBuilder = require('fat-js').RPCBuilder
+let rpc = new RPCBuilder()
+    .host('fatnode.mysite.com')
+    .port(1234)
+    .version('v0')
```

Comment:
What is your intention there? If somebody pass v0 you will always provide support for the v0 API? That may be risky to commit that 1 version of the lib will support multiple versions of the protocol, and people usually don't do that.

Reply:
Thanks, this is a good point. If this is exposed it should probably be more structured, regardless